src/pages/postgraphile/security.md

```diff
@@ -139,12 +139,20 @@ const authLink = setContext((_, { headers }) => {
   // get the authentication token from wherever you store it
   const token = getJWTToken();
   // return the headers to the context so httpLink can read them
+  if (token) {
+    return {
+      headers: {
+        ...headers,
+        authorization: `Bearer ${token}`,
+      },
+    }
+  }
   return {
     headers: {
       ...headers,
-      authorization: token ? `Bearer ${token}` : "",
+      // if no jwt, we don't want to pass the auth header at all
```

Comment:
I'd rather solve this inline than adding a separate headers object above. We can do it like this, I think:

```suggestion
      // Only pass the authorization header if we have a JWT
      ...(token ? { authorization: `Bearer ${token}` } : null),
```